class AuthError(Exception):
    pass


class OApiError(Exception):
    pass
